docker-compose.yml

```diff
@@ -27,8 +27,10 @@ services:
                     }
                 }
             }'
-            PGHOST: postgres
-            PGUSER: xpub
+            PGHOST: 
```

Comment:
there's also `PGPORT`:
https://github.com/elifesciences/elife-xpub-formula/blob/0e179a71133ca309e9aeafe34d60c5cfc6c4ac47/salt/elife-xpub/init.sls#L51

Reply:
It's very unlikely to change from the default, but RDS seems to provide it so you never know if they may change it one day.